geode-book/master_middleman/source/subnavs/geode-subnav.erb

```diff
@@ -2197,6 +2197,9 @@ gfsh</a>
                             <li>
                                 <a href="/docs/guide/<%=vars.product_version_nodot%>/tools_modules/redis_api_for_geode.html#advantages-of-<%=vars.product_name_lowercase%>-over-redis">Advantages of <%=vars.product_name%> over Redis</a>
```

Comment:
Not part of this PR, but a potential problem. We use template variables to allow portability to other products that use the geode open-source repo. Use in section names (such as "Advantages of <%=vars.product_name%>..." follows this intention. However, use in target ids (such as "#advantages-of-<%=vars.product_name_lowercase%>-over-redis") defeats the purpose, as the links are almost certainly guaranteed to fail in other contexts. The hash tags should be genericized.

Reply:
I went ahead and updated it.